dockerfiles/ink-ci-linux/README.md

```diff
@@ -45,7 +45,7 @@ We always try to use the [latest possible](https://rust-lang.github.io/rustup-co
 ```yaml
 test-ink:
     stage: test
-        image: paritytech/ink-ci-linux:latest
+        image: paritytech/ink-ci-linux:production
```

Comment:
What does this do?

Reply:
`latest` tag is no longer pushed for a while.